integration/duckdb/CMakeLists.txt

```diff
@@ -97,18 +105,27 @@ if(LANCE_BUILD_PYTORCH)
 endif()
 
 include_directories(src)
-set(LANCE_EXT_SOURCES
-    src/lance/duckdb/lance-extension.cc
-    src/lance/duckdb/list_functions.cc
-    src/lance/duckdb/list_functions.h
-    src/lance/duckdb/vector_functions.cc
-    src/lance/duckdb/vector_functions.h
-    src/lance/duckdb/ml/catalog.cc
-    src/lance/duckdb/ml/catalog.h
-    src/lance/duckdb/ml/pytorch.cc
-    src/lance/duckdb/ml/pytorch.h
-    src/lance/duckdb/ml/functions.cc
-    src/lance/duckdb/ml/functions.h)
+if(LANCE_BUILD_PYTORCH)
+  set(LANCE_EXT_SOURCES
+      src/lance/duckdb/lance-extension.cc
+      src/lance/duckdb/list_functions.cc
+      src/lance/duckdb/list_functions.h
+      src/lance/duckdb/vector_functions.cc
+      src/lance/duckdb/vector_functions.h
+      src/lance/duckdb/ml/catalog.cc
+      src/lance/duckdb/ml/catalog.h
```

Comment:
set common `LANCE_EXT_SOURCES`, and append pytorch related files within `if .. end` ?

Reply:
fixed